src/op_mode/ntp.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2024 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import sys
+import typing
+
+import vyos.opmode
+from vyos.utils.process import cmd
+
+def show_summary(raw: bool, command: typing.Optional[str]):
+    output = cmd(f"chronyc '{command}'")
+
+    # Does not return raw dict as implemented and here
+    # to satisfy run(module) function in opmode.py
+    if raw:
```

Comment:
It has to return raw values.
The `raw` format is required 
chronyc has the `-c` option `Enable CSV format`. For example, it could be used as values of the dictionary

Reply:
There's `csv` module that can convert it to a dict quite easily, so it's not a time-consuming problem.

@Giggum Raw outputs are used by the HTTP API, which was the main motivation for the new format (other than just not having to write option parsing by hand for CLI scripts).

Reply:
Ok, will look into this.
FYI @sever-severm, I've resolved the other issue of checking that the service exists.

Reply:
@sever-sever raw format is implemented. If you're good let me know to remove Draft from PR